fw/http.c

```diff
@@ -3581,8 +3572,19 @@ tfw_h2_adjust_req(TfwHttpReq *req)
 	}
 
 	r |= tfw_msg_write(&it, &h_via);
-	if (need_cl)
+	if (need_cl) {
+		h_cl = (TfwStr) {
+			.chunks = (TfwStr []) {
+				{ .data = "Content-Length", .len = SLEN("Content-Length") },
+				{ .data = S_DLM, .len = SLEN(S_DLM) },
+				{ .data = cl_data, .len = cl_len },
+				{ .data = S_CRLF, .len = SLEN(S_CRLF) }
+			},
+			.len = cl_len,
```

Comment:
Oops, now the both the length are initialized into `cl_len`, while they were different in the original code..